src/DataXml/Utd820/ON_NSCHFDOPPOK_UserContract_820_05_01_01.cs

```diff
@@ -0,0 +1,649 @@
+
+// 
+// This source code was auto-generated by xsd, Version=4.6.1055.0.
+// 
+namespace Diadoc.Api.DataXml.Utd820 {
+    using System.Xml.Serialization;
+    
+    
+    /// <remarks/>
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("xsd", "4.6.1055.0")]
+    [System.SerializableAttribute()]
+    [System.Diagnostics.DebuggerStepThroughAttribute()]
+    [System.ComponentModel.DesignerCategoryAttribute("code")]
+    [System.Xml.Serialization.XmlTypeAttribute(AnonymousType=true)]
+    [System.Xml.Serialization.XmlRootAttribute(Namespace="", IsNullable=false)]
+    public partial class UniversalTransferDocumentBuyerTitle {
+        
+        private Employee employeeField;
+        
+        private OtherIssuer otherIssuerField;
+        
+        private AdditionalInfoId additionalInfoIdField;
+        
+        private object[] signersField;
+        
+        private UniversalTransferDocumentBuyerTitleContentOperCode contentOperCodeField;
+        
+        private UniversalTransferDocumentBuyerTitleBuyerInfoCircumPublicProc buyerInfoCircumPublicProcField;
+        
+        private string documentCreatorField;
+        
+        private string documentCreatorBaseField;
+        
+        private string operationCodeField;
+        
+        private string operationContentField;
+        
+        private string acceptanceDateField;
+        
+        /// <remarks/>
+        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
+        public Employee Employee {
+            get {
+                return this.employeeField;
+            }
+            set {
+                this.employeeField = value;
+            }
+        }
+        
+        /// <remarks/>
+        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
+        public OtherIssuer OtherIssuer {
+            get {
+                return this.otherIssuerField;
+            }
+            set {
+                this.otherIssuerField = value;
+            }
+        }
+        
+        /// <remarks/>
+        [System.Xml.Serialization.XmlElementAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
+        public AdditionalInfoId AdditionalInfoId {
+            get {
+                return this.additionalInfoIdField;
+            }
+            set {
+                this.additionalInfoIdField = value;
+            }
+        }
+        
+        /// <remarks/>
+        [System.Xml.Serialization.XmlArrayAttribute(Form=System.Xml.Schema.XmlSchemaForm.Unqualified)]
+        [System.Xml.Serialization.XmlArrayItemAttribute("SignerDetails", typeof(ExtendedSignerDetails_BuyerTitle820), Form=System.Xml.Schema.XmlSchemaForm.Unqualified, IsNullable=false)]
+        [System.Xml.Serialization.XmlArrayItemAttribute(typeof(SignerReference), Form=System.Xml.Schema.XmlSchemaForm.Unqualified, IsNullable=false)]
+        public object[] Signers {
```

Comment:
Consider provide extension methods (or props?) to set `ExtendedSignerDetails_BuyerTitle820` and `SignerReference` typesafely. It may be nice for end users.

Reply:
Made a couple of methods